### ISSUE TITLE
Dual boot on bare metal

### ISSUE THREAD

**Issue by kaievns** [siderolabs/talos#9592]
## Feature Request
Add support for dual boot option for the bare metal setups

### Description
Say I have a windows PC in a homelab that I would like to use as a Talos node without wiping the windows off. What I would like is to point Talos to an empty space on my HDD and install it with GRUB boot loader like normal desktop linux systems.

I'm happy for it to default to Talos, I just want to have an option to boot back to windows now and then, that's all

**Comment by packeteer** [comment]
I dont think this would be considered, as it would require a deep re-architect of the way Talos works.

For testing purposes, you'd be better of running under Docker

**Comment by rothgar** [comment]
Sorry but we have no plans to support a dual-boot setup. You can install to a USB/external drive and boot from the   drive but doing this via a bootloader isn't planned.
